cfbs/validate.py

```diff
@@ -84,7 +84,7 @@ def validate_version(name, modules):
             raise CFBSIndexException(name, "Missing required attribute 'version'")
         if type(modules[name]["version"]) != str:
             raise CFBSIndexException(name, "'version' must be of type string")
-        regex = r"(0|[1-9][0-9]*).(0|[1-9][0-9]*).(0|[1-9][0-9]*)"
+        regex = r"(0|[1-9][0-9]*).(0|[1-9][0-9]*).(0|[1-9][0-9]*)(-([0-9]*)?)?"
```

Comment:
The `*` should be `+` and the first `?` should be dropped, AFAICT. We don't want versions like `1.0.0-`, there must be at least one digit.

Reply:
sure, makes sense.

Reply:
I also converted the `.` to `\.` so that you something like `3.1-20` doesn't validate. :)